types/react-dom/client.d.ts

```diff
@@ -32,9 +32,9 @@ export interface Root {
 }
 
 /**
- * Replaces `ReactDOM.render` when the `.render` method is called and enables Concurrent Mode.
+ * createRoot lets you create a root to display React components inside a browser DOM node.
```

Comment:
Just copy&paste from reference docs. Didn't add some to the one for `document` in Canary since we're missing docs for that.